src/components/video-editor/VideoEditor.tsx

```
@@ -1856,18 +2035,50 @@ export default function VideoEditor() {
 	}, [videoPath, duration, effectiveCursorTelemetry, loopCursor, zoomRegions.length]);
 
+	const initAudioContext = useCallback(() => {
+		if (audioContextRef.current) {
+			if (audioContextRef.current.state === 'suspended') {
+				audioContextRef.current.resume().then(() => setIsAudioEngineReady(true));
+			}
+			return;
+		}
+		
+		try {
+			const ctx = new (window.AudioContext || (window as any).webkitAudioContext)();
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)